samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -65,43 +65,35 @@ public DocumentTranslationClient(string endpoint, AzureKeyCredential credential,
         /// <param name="documentId"> Format - uuid.  The document id. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
+        /// Additional information can be found in the service REST API documentation:
+        /// https://docs.microsoft.com/rest/api/AzureAIDocumentTranslation/DocumentTranslation/GetDocumentStatus
```

Comment:
This rest API link doesn't work.

Reply:
This link is generated according to the doc link format see https://github.com/Azure/autorest.csharp/issues/2253. If the service put the doc in a link which does not comply the format, it need add 'externalDocs' in the swagger.

Reply:
The generated output doesn't need to be a correct link in the autorest.csharp repo, but it will need to be a correct link in the azure-sdk-for-net repo.  @pallavit, who on our team owns the DocumentTranslation library and would be responsible for making this update to their swagger file?  @lmazuel FYI